### PR TITLE
[GUILD-2599] - Email allowlist improvements

### PR DESCRIPTION
**Pull request by cs-balazs**
[Linear](https://linear.app/guildxyz/issue/GUILD-2599/smaller-email-allowlist-improvements) - [Discord](https://discord.com/channels/697041998728659035/1221827375910621286/1222193169320382606)

- Show ALLOWLIST_EMAIL as an option under the email category as well
- Show connect button on requirement card
  - For this, I've moved some code to a separate component
- Change EMAIL colorScheme to blue

### CODE REVIEW

File: src/requirements/Allowlist/AllowlistRequirement.tsx [hunk 1]
```diff
@@ -24,10 +25,14 @@ const AllowlistRequirement = ({ ...rest }: RequirementProps): JSX.Element => {
     <Requirement
       image={<Icon as={ListPlus} boxSize={6} />}
       footer={
-        hideAllowlist && (
-          <Text color="gray" fontSize="xs" fontWeight="normal">
-            {`Allowlisted ${isEmail ? " email" : ""} addresses are hidden`}
-          </Text>
+        isEmail ? (
```
Comment (BrickheadJohnny) [inline src/requirements/Allowlist/AllowlistRequirement.tsx:28]: I this way the "allowlisted ... addresses are hidden" test wont show up, is that intended? 👀

**Review by BrickheadJohnny — COMMENTED** [review]
Left one minor comment, otherwise LGTM! 🚀

File: src/requirements/Allowlist/AllowlistRequirement.tsx (hunk 1 above)
Comment (cs-balazs) [reply src/requirements/Allowlist/AllowlistRequirement.tsx:36]: Oh, right, changed in https://github.com/guildxyz/guild.xyz/pull/1140/commits/95dc304384882c6cefe24fc851b1cb3ab5bd2c70 👍

**Comment by cs-balazs** [comment]
https://github.com/guildxyz/guild.xyz/pull/1140/commits/6d14e0a15e8693552ad072ae426e6b929edc5409 is intended to fix [GUILD-2570](https://linear.app/guildxyz/issue/GUILD-2570/cant-save-allowlist-requirement)

**Pull request merged**